src/python/include/Persistent_cohomology_interface.h

```diff
@@ -73,6 +72,54 @@ persistent_cohomology::Persistent_cohomology<FilteredComplex, persistent_cohomol
     return persistence;
   }
 
+  int top_dimensional_coface(int splx){
```

Comment:
Could we move this utility function to the main classes, not hidden in the python binding code? With a more explicit name.

Reply:
You mean in `Persistent_cohomology.h`? Or somewhere else?

Reply:
In src/Bitmap_cubical_complex/include/gudhi/something. It doesn't depend on persistence.

Reply:
Sure. Should be done now.